Formula/a/atf.rb

```diff
@@ -5,10 +5,12 @@ class Atf < Formula
   sha256 "a64e2427d021297f25b3f2e1798f8ec4dc3061ffb01a1cd3f66cc4cee486b10f"
   license "BSD-2-Clause"
   head "https://github.com/freebsd/atf.git", branch: "master"
-
+  # Upstream creates pre-released tags as part of their release process, so use
+  # the `GithubLatest` strategy to pick up the latest non-release version prior
+  # to the official release.
   livecheck do
     url :stable
-    regex(/^atf[._-]?v?(\d+(?:\.\d+)+)$/i)
+    strategy :github_latest
```

Comment:
You can split this changes to another PR, this doesn't require build process.

Reply:
Ok.

Reply:
It is, it has nothing to do with the other changes. That makes it critical for them to be split

Reply:
Done: #260356